boring/src/asn1.rs

```diff
@@ -504,22 +505,36 @@ impl Asn1ObjectRef {
     pub fn nid(&self) -> Nid {
         unsafe { Nid::from_raw(ffi::OBJ_obj2nid(self.as_ptr())) }
     }
-}
 
-impl fmt::Display for Asn1ObjectRef {
-    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+    /// Returns the numerical string OID of this object.
+    ///
+    /// This corresponds to [`OBJ_obj2txt`] with `no_name = 1`.
+    ///
+    /// [`OBJ_obj2txt`]: https://www.openssl.org/docs/man1.1.1/man3/OBJ_txt2obj.html
+    pub fn oid_string(&self) -> Result<String, Utf8Error> {
+        self.to_text(false)
```

Comment:
I think this should be `true`? 
I suspect it slipped by because when there is no textual name for an object false/true both return the numerical string 
```suggestion
        self.to_text(true)
```